pkg/network/networker.go

```diff
@@ -285,7 +286,7 @@ func (n networker) ZDBPrepare(hw net.HardwareAddr) (string, error) {
 
 	}
 
-	return netNSName, n.createMacVlan(ZDBIface, hw, ips, routes, netNs)
+	return netNSName, n.createMacVlan(ZDBIface, types.YggBridge, hw, ips, routes, netNs)
```

Comment:
The yggdrasil macvlan for zdbs should be installed next to the already existing macvlan, not replace it

Reply:
Ah right, so zdb will has both as well. right. This means we need another interface in the zdb container ?

Reply:
indeed, everything with ygg IP will have a dedicated interface for that. Later when a vm can have an ygg IP, it will also create a dedicated tap iface on the ygg bridge next to the existing ones. That way traffic flow is nicely separated.